src/components/shared/MarkerSettings.js

```diff
@@ -35,13 +36,18 @@ class MarkerSettingsImpl extends PureComponent<Props> {
     return (
       <div className="markerSettings">
         <div className="markerSettingsSpacer" />
-        <PanelSearch
-          className="markerSettingsSearchField"
-          label="Filter Markers: "
-          title="Only display markers that match a certain name"
-          currentSearchString={searchString}
-          onSearch={this._onSearch}
-        />
+        <Localized
+          id="MarkerSettings--panel-search"
+          attrs={{ label: true, title: true }}
+        >
+          <PanelSearch
+            className="markerSettingsSearchField"
+            label="Filter Markers: "
```

Comment:
What about removing the final space in this label? This might make things clearer to a reader.

Same in `NetworkSettings.js`

Reply:
Yeah, removed them.